src/index/IndexImpl.cpp

```diff
@@ -1083,11 +1089,13 @@ void IndexImpl::readIndexBuilderSettingsFromFile() {
    * locale setting.
    */
 
-  LOG(INFO) << "You specified \"locale = " << lang << "_" << country << "\" "
-            << "and \"ignore-punctuation = " << ignorePunctuation << "\""
-            << std::endl;
+  LOG(INFO) << "You specified \"locale = " << configVariables.localLang_ << "_"
+            << configVariables.localCountry_ << "\" "
+            << "and \"ignore-punctuation = "
+            << configVariables.localIgnorePunctuation_ << "\"" << std::endl;
```

Comment:
I think we can delete those lines now that we have the automatic printing of the configuration.

Reply:
Done, see [76b5747](https://github.com/ad-freiburg/qlever/pull/1041/commits/76b574703a9a7fe7f260e5a4c0ebcdc9544f710f).